src/site/stages/build/drupal/graphql/paragraph-fragments/alert.paragraph.graphql.js

```diff
@@ -0,0 +1,59 @@
+/*
+*
+* A reusable or non-reusable alert, either "information status" or "warning status".
+*
+ */
+module.exports = `
+  fragment alertParagraph on ParagraphAlert {      
+      fieldAlertType
+      fieldAlertHeading
+      fieldVaParagraphs {
+        entity {
+          ... on ParagraphWysiwyg {
+            entityId
+            entityBundle
+            fieldWysiwyg {
+              processed
+            }
+          }
+          ... on ParagraphExpandableText {
+            entityId
+            entityBundle
+            fieldWysiwyg {
+              processed
+            }
+            fieldTextExpander
+          }
+        }
+      }
+      fieldAlertBlockReference {
+        entity {
+          ... on BlockContentAlert {
+            fieldAlertTitle
+            fieldHelpText
+            fieldAlertType
+            fieldReusability
+            fieldAlertContent {
+              entity {
+                ... on ParagraphWysiwyg {
+                  entityId
+                  entityBundle
+                  fieldWysiwyg {
+                    processed
+                  }
+                }
+                ... on ParagraphExpandableText {
+                  entityId
+                  entityBundle
+                  fieldWysiwyg {
+                    processed
+                  }
+                  fieldTextExpander
+                }
+              }
```

Comment:
It doesn't look like we can do it without modifying the exports, but we can potentially share these fields with `src/site/stages/build/drupal/graphql/block-fragments/alert.block.graphql.js`. Up to you though.